### PR TITLE
Update switch.md

### PR DESCRIPTION
**Pull request by ghost**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/language-reference/keywords/switch.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Starting with C# 7.0, the match expression can be any non-null expression.
  
 ## The switch section
  
- A `switch` statement includes one or more switch sections. Each switch section contains one or more *case labels* followed by one or more statements. The following example shows a simple `switch` statement that has three switch sections. Each switch section has one case label, such as `case 1:`, and two statements.
+ A `switch` statement includes one or more switch sections. Each switch section contains one or more *switch labels* (either a case or default label) followed by one or more statements. The `switch` statement may include at most one default label placed in any switch section. The following example shows a simple `switch` statement that has three switch sections, each containing two statements. The second switch section contains the `case 2:` and `case 3:` labels.
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/switch.md]: Let's revert the one change here from *case labels* to *switch labels*.

The only place where the term "switch label" appears is in the grammar. Everywhere else I looked in the C# spec, it's referred to as a "case label".

**Review by BillWagner — CHANGES_REQUESTED** [review]
Overall, this looks great @johnnyagerard I like the clarifications you've made.

I had one request to roll back one change. I think it would lead to confusion among readers.

**Comment by johnnyagerard** [comment]
I just added a commit.

I don't know how to modify my first commit.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
@johnnyagerard You did the right thing by adding a new commit. That automatically updates the PR. I've looked at the changes, and I'll :shipit: as soon as the build finishes.

Thanks again.

**Comment by BillWagner** [comment]
The build appears stuck. closing and reopening to start a new build.

**Pull request merged**